src/realm/exec/realm_trawler.cpp

```diff
@@ -606,16 +606,20 @@ void Table::print_columns(const Group& group) const
 {
     std::cout << "        <" << m_table_type << ">\n";
     for (unsigned i = 0; i < m_column_names.size(); i++) {
-        auto type = realm::ColumnType(m_column_types.get_val(i) & 0xFFFF);
+        auto type = m_column_types.get_val(i) & 0xFFFF;
         auto attr = realm::ColumnAttr(m_column_attributes.get_val(i));
         std::string type_str;
         realm::ColKey col_key;
         if (this->m_column_colkeys.valid()) {
             // core6
             col_key = realm::ColKey(m_column_colkeys.get_val(i));
         }
+        if (type == 13) {
+            // type_LinkList == 13
+            type = 12;
+        }
 
-        if (type == realm::col_type_Link) {
+        if (type == 12) {
```

Comment:
nit: but isn't it easier to just `if (type == 12 || type == 13)`?

Reply:
Then the call in line 635 would not work